cloudify_aws/cloudformation/resources/stack.py

```diff
@@ -145,6 +145,16 @@ def test(_value):
         tested_value = test(value)
         ctx.instance.runtime_properties[key] = tested_value
 
+    # Special handling for outputs: they're provided by the stack
+    # as a list of key-value pairs, which makes it impossible to
+    # use them via intrinsic functions. So, create a dictionary out
+    # of them.
+    if 'Outputs' in props:
+        outputs_as_dict = {}
+        for output in props['Outputs']:
+            outputs_as_dict[output['OutputKey']] = output['OutputValue']
+        ctx.instance.runtime_properties['outputs_as_dict'] = outputs_as_dict
```

Comment:
I'm not sure that I really like the key "outputs_as_dict". Maybe there is another term like `outputs_dictionary` or `outputs_items`. I think the latter.

Reply:
@EarthmanT will be resolved in a followup PR